docs/walkthroughs/connecting-device-to-service.adoc

```diff
@@ -0,0 +1,48 @@
+[[mobile-core-public-ip]]
+= Mobile Core installation with public IP
+To start openshift cluster for local development and connect your device to same network you need to specify public IP of
+your laptop or server you want openshift being installed and provide following parameters
+```
+Use public IP yes/no?: _yes_
+Cluster IP local/public: <your-eth0-ip>
+```
+image::images/mobile-core-public-ip-setup.png[openshift-mobile-core-public-ip-setup]
+Find IP address of your local machine you can with this command ```ifconfig | grep inet```.
+Output should look like this
+```
+	inet 127.0.0.1 netmask 0xff000000
+	inet 10.40.3.70 netmask 0xfffffe00 broadcast 10.40.3.255
+```
+When installation is finished you should get same result as on image below
+
+image::images/mobile-core-public-ip-success.png[openshift-mobile-core-public-ip-success]
+Web console URL should contain your public IP address in my case it is ```10.40.3.70```
+
+[[mobile-core-services]]
```

Comment:
Not sure we need the bit below this. I think if we end by saying now with your mobile device connected to the same network as your local machine, you should be able to make requests to any mobile services you have provisioned.